metoncofit/DEA/CCLE_DEA.R

```diff
@@ -0,0 +1,71 @@
+#### imports ####
+ccle = import("~jennadiegel/Documents/SURE/DES/CCLE_data.csv")
+
+gene_names <- ccle[,2]
+wt_vals <- ccle[ ,3:4]
+mut_vals <- ccle[ ,-(1:4)]
+
+#### analysis ####
+
+#fill in NA values with 0 - database excluded 0 values
+wt_vals[is.na(wt_vals)] <- 0
+mut_vals[is.na(mut_vals)] <- 0
+
+#calculate WT mean from 2 WT cell lines
+WT_mean <- (mut_vals[,1] + mut_vals[,2])/2
+
+
+#determine fold change for all cell lines
+fold_changes <- list()
+fold_changes_log2 <- list()
+noninf_gene_names <- list()
+inf_gene_names <- list()
+z_scores <- list()
+p_values <- list()
+all_data <- list()
+i = 1
+for (i in 1:33) {
```

Comment:
I would state exactly why the end of the for loop is 33 - not sure why that is. Is 33 the number of columns in the frame?